docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,284 @@
+.. _IFV:
 
-Visualization
-=============
+IDAES Flowsheet Visualizer
+===========================
 
-.. warning::
-    The visualization tool is still in active development and we
-    hope to improve on it in future releases. Please use its
-    functionality at your own discretion.
+.. contents::
+    :depth: 2
 
-Overview
+Concepts
 --------
+The IDAES Flowsheet Visualizer, or IFV for short, is a graphical user interface for viewing an IDAES flowsheet.
+It uses the visual language of traditional process engineering diagrams to display
+the components of a given flowsheet,
+their connections, and values associated with unit models and streams. While the changes made from the UI do
+not affect the underlying model, changes in the flowsheet -- e.g., from Python code or from solvers updating the
+stream values -- are immediately available to be displayed in the IFV. The IFV does not require running a
+separate program, as its functionality is embedded in the invoking Python code (see the `Client/server architecture <ifv-architecture>`_ for details).
+Starting and stopping the IFV is fast and does not consume many resources, a fact which we hope encourages
+its frequent use.
+
```

Comment:
I think that we need to mention that if they restart the Jupyter notebook kernel they lose the ability to save I know that you mention it in the advanced documentation but it needs to be more prominent.

Reply:
yes, agree. It has been moved into the docstring for the 'visualize()' function, so will appear near the top of the "User Guide" section (and in the code!)